clang/lib/Driver/Driver.cpp

```diff
@@ -5683,7 +5732,22 @@ class OffloadingActionBuilder final {
           Arg *SYCLTargetsValues = SYCLTargets ? SYCLTargets : SYCLLinkTargets;
           // Fill SYCLTripleList
           llvm::StringMap<StringRef> FoundNormalizedTriples;
-          for (const char *Val : SYCLTargetsValues->getValues()) {
+          for (StringRef Val : SYCLTargetsValues->getValues()) {
+            StringRef Val_t(Val);
+            // Handle target specifications that resemble 'intel_gpu_*' here.
+            // These are 'spir64_gen' based.
+            if (Val.startswith("intel_gpu_")) {
```

Comment:
This code is used 3 times and could be a function or lambda.
`"intel_gpu_"` is used even more. Perhaps some `auto constexpr` variable somewhere?